src/StaticLS/IDE/CodeActions/InsertMissingMethods.hs

```diff
@@ -0,0 +1,15 @@
+module StaticLS.IDE.CodeActions.InsertMissingMethods where
+
+import StaticLS.IDE.CodeActions.Utils
+
+import Data.Text qualified as T
+import Language.LSP.Protocol.Types qualified as LSP
+
+codeAction :: LSP.TextDocumentIdentifier -> LSP.Diagnostic -> [T.Text] -> LSP.CodeAction
+codeAction = insertMissingMethods
+
+insertMissingMethods :: LSP.TextDocumentIdentifier -> LSP.Diagnostic -> [T.Text] -> LSP.CodeAction
+insertMissingMethods tdi diag methods =
```

Comment:
If a GHC error (or warning) says your class instance is missing methods, this function will create a quick fix that will insert the missing methods for you.